front-end/src/components/FileStorage.js

```diff
@@ -103,6 +119,23 @@ const FileStorage = ({ files, selectedFiles, setSelectedFiles }) => {
                     folderRenderer={CustomFolderRenderer}
                 />
             </div>
+            <div className="buttonFlexBox">
+        <button className="submitbutton" onClick={() => {setDisplayPage(1)}}>Back</button>
+        <button className="submitbutton" onClick={() => {
+          // OnClick, it should get the selected files from the file storage component
+          //getSelectedFiles();
+          console.log("selected files")
+          console.log(selectedFiles)
+          getHeaders(selectedFiles)
+          // Instead of this function, go through 
+          if (graphType === "XYColour") {
```

Comment:
This, along with graphType and setDimensions are outdated, should be removed

Reply:
as the author of this issue i believe that i understand the purpose better then anybody